assets/prefabs/structures/types/dungeonTownBuilding.prefab

```diff
@@ -2,6 +2,6 @@
     "StructureTemplateType": {},
     "DisplayName": {
         "name": "Dungeon Town Building",
-        "description": "A building in a dungeon town. Region for building: {min: [-4, 1, 2], size: [10, 9, 7]}"
+        "description": "A building in a dungeon town. Region for building: {min: [-4, 1, 2], size: [9, 9, 7]}"
```

Comment:
@flo Have you tested by actually using a block of 10x9x8 or by measuring the dimensions of town? I believe there is a hitch in the code that causes only a 9x9x7 block to appear